web/static/js/components/surveys/SurveyStatus.jsx

```diff
@@ -215,8 +211,10 @@ class SurveyStatus extends PureComponent {
 }
 
 const mapStateToProps = (state) => {
+  const { settings } = state.userSettings
   return {
-    timezones: state.timezones
+    timezones: state.timezones,
+    language: settings && settings.language || 'en'
```

Comment:
```suggestion
    language: i18next.language
```

I would expect to use the current language from i18n and not duplicate the logic of how the current language is defined and which one is the default.

Does that do not work?

It can also be called within `nextCallDescription`, there is no need to make it a prop since it's a global setting and the `t` func already depends on `i18next`